projects/freertos/mctp_cmd_task.c

```diff
@@ -19,6 +19,7 @@ static void mctp_cmd_task_loop (void *data)
 
 	while (1) {
 		cmd_channel_receive_and_process (task->channel, task->mctp, -1);
+		vTaskDelay(1000);
```

Comment:
Why is a delay necessary here?

Reply:
The idea is to change the task to the blocked state to allow other tasks to run.

Reply:
cmd_channel_receive_and_process will call cmd_channel.receive_packet, which will block until a packet is available from the channel.  I see now that the comment on cmd_channel does not make this clear, so I'll plan to update it.

Given that, there is no need to add artificial delay here, and we wouldn't want to prevent another command from being executed until a full second after a previous command, either.